astarte-device-conf/.env

```diff
@@ -1,6 +1,13 @@
+# general config
+ASTARTE_CONNECTION=mqtt
+ASTARTE_STORE_DIRECTORY=/tmp/stream-rust-test/store/
+
+# MQTT connection to astarte configs
 ASTARTE_REALM=test
 ASTARTE_DEVICE_ID=2TBn-jNESuuHamE2Zo1anA
-ASTARTE_CREDENTIALS_SECRET=3wVjV6A+iAWmTLopPb5A3lRFCVsKXPhNxFDxw2Aunb0=
+ASTARTE_CREDENTIALS_SECRET=QdjYP9UyeuTUd3YH+3Xjd2BPQP79bxU4ewtJS62m/YM=
 ASTARTE_PAIRING_URL=http://api.astarte.localhost/pairing
```

Comment:
This file is already tracked by git, so you can remove it from git history or squash the commit from the first time it was added.